portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,32 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comments */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments;
+
+  @Override
+  public void init(){
+    comments = new ArrayList<String>();
+    comments.add("Hola amigo");
+    comments.add("This is a comment");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Might be good to get a Javadoc on this since the method name doesn't fully explain what you're getting: https://google.github.io/styleguide/javaguide.html#s7-javadoc

Reply:
Great! Sorry for those mistakes, I didn't use to look up style guides but now I know they are very important. I will keep improving on that 😃